src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormFieldValue.java

```diff
@@ -137,6 +137,11 @@ public String getType() {
         return type;
     }
 
+    public String getValue() {
```

Comment:
What's the difference between this method and `getStringValue`?

Reply:
Those two methods get the same result but ObjectMapper needs (by default) a getter which have the same name of the properties of the class that you want to fill. Before my modification we had a property named `value` with a getter named `getStringValue` so ObjectMapper can not do the mapping automatically.

Reply:
Good to know, thank you :D